app/view.py

```diff
@@ -8,13 +7,19 @@
 import pandas as pd
 import plotly.express as px
 import plotly.graph_objects as go
+from plotly.graph_objs import Line
+from plotly.graph_objs.scatter.marker import Line
```

Comment:
Warum beide Line imports hier? Das Obere sollte ja überschrieben werden damit.

Reply:
Für Simulierte Daten hab ich die Deutschlandkarte nicht getestet, Max hat das mal gemacht.

Tqdm ist einfach dazu da um einen Fortschrittsbalken anzuzeigen bei der Berechnung der Deutschlandkarte, damit man weiß wie lange es noch dauert und nicht vorher abbricht

Reply:
> Für Simulierte Daten hab ich die Deutschlandkarte nicht getestet, Max hat das mal gemacht.
> 
> Tqdm ist einfach dazu da um einen Fortschrittsbalken anzuzeigen bei der Berechnung der Deutschlandkarte, damit man weiß wie lange es noch dauert und nicht vorher abbricht

Ok

Reply:
> Beim Plot hab ich ein DeprecationWarning bekommen, Plot hat zwar Funktioniert, nach google sollte ich die beiden importieren, dann war die Warnung nicht mehr da, kann aber glaub ich auch weglassen.

Was war denn deprecated? In dem Fall sollten wir eher die überholte Funktion rausnehmen und durch die neue ersetzen. Das Deprecation Warning schlägt ja vermutlich was vor?